test/e2e/util/utils.go

```diff
@@ -79,7 +79,7 @@ var (
 	TestLogDir         = os.Getenv("TEST_LOGDIR")
 
 	ParsedImages        *Images
-	Timeout             = time.Minute * 5
```

Comment:
Why the increase?

Reply:
some of the tests were failing consistently due to timeout. I'll check again with the new changes to see if it works with 5 minutes again